contact_us.php

```diff
@@ -0,0 +1,29 @@
+<?php session_start(); ?>
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <link href="css/index.css" type="text/css" rel="stylesheet"/>
+    <title>Contact Us</title>
+  </head>
+  <body>
+    <h1 class="logo">evem</h1>
+    <div class="nav">
+        <ul>
+          <li><a href="welcome.php">Home</a></li>
+          <li><a href="http://localhost/evem/php/event_list.php">Events</a></li>
```

Comment:
```suggestion
          <li><a href="php/event_list.php">Events</a></li>
```
don't specify absolute links....make it relative to your root folder `evem`